libpod/container_config.go

```diff
@@ -269,24 +271,24 @@ type ContainerNetworkConfig struct {
 	// Hosts to add in container
 	// Will be appended to host's host file
 	HostAdd []string `json:"hostsAdd,omitempty"`
-	// Network names (CNI) to add container to. Empty to use default network.
+	// Network names with the network specific options.
+	// Please note that these can be altered at runtime. The actual list is
+	// stored in the DB and should be retrieved from there via c.networks()
+	// this value is only used for container create.
+	// Added in podman 4.0, previously NetworksDeprecated was used. Make
+	// sure to not change the json tags.
+	Networks map[string]types.PerNetworkOptions `json:"newNetworks,omitempty"`
```

Comment:
Do we need this if we always use `c.networks()` instead?

Reply:
we need it in order to add networks during container creation with `WithNetNS()`

Reply:
the other option would be to add networks as function parameter for all container create function calls but this would make the diff even bigger

Reply:
Ahhh. OK, I can see how this works.